.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       run: tox
 
     - name: Run coverage
-      if: matrix.python-version == '3.8' && matrix.toxenv == 'django40'
+      if: matrix.python-version == '3.8' && matrix.toxenv == 'django42'
```

Comment:
I changed `tox.ini` to use Django 4.2 in https://github.com/edx/learning-assistant/pull/93 and forgot to update this line.